### PR TITLE
changed the caption of my md file

### LINKED ISSUE #5: Improve the README
## Do you have a favorite Git Resource?
Add it to the README!
 If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by pfekrati**
this change was simple and just updated the place holder line.
this PR fixes #5 

# Done
- this
-  that

## to do 
- [ ] number one
- [ ] number two

cc @pfekrati-demo

### CODE REVIEW

File: _slides/02-pfekrati.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (pfekrati-demo) [inline _slides/02-pfekrati.md:5]: why is there a blank line

Comment (pfekrati) [reply _slides/02-pfekrati.md:5]: formatting?

File: _slides/02-pfekrati.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-Caption here
+Hello!!
```
Comment (pfekrati-demo) [inline _slides/02-pfekrati.md:6]:
```suggestions
hello all!!
```

Comment (pfekrati-demo) [reply _slides/02-pfekrati.md:6]:
```suggestion
hello all!!
```

**Review by pfekrati-demo — APPROVED** [review]
this looks good

**Pull request merged**